README.md

```diff
@@ -113,15 +87,37 @@ with that major/minor version.
 
 Client library developers are not expected to monitor the docs repo for
 spec fixes that occur after the release tag. If a given spec fix needs
-to be made to client libraries at that time, on merging the pr to
-`master` you should open a github issue in each individual client lib
+to be made to client libraries at that time, on merging the PR to
+`master` you should open a GitHub issue in each individual client lib
 repo to request that. If this is not done, it is not mandatory for the
 fix to be incorporated until the next spec release.
 
-When updating a client lib to a spec version, client lib devs should
+When updating a client lib to a spec version, client lib developers should
 work from a diff from the tag of the previous release, so as to
 incorporate all changes since that tag.
 
+## Deploying to website (www.ably.io/documentation)
+
+The website consumes this repo through a ruby gem. The gem points at the `master` branch and saves the revision as a version. To release to `/documentation`, follow these steps:
```

Comment:
Minor point - ruby is referred to as Ruby elsewhere on this page

Reply:
```suggestion
The website consumes this repo through a Ruby gem. The gem points at the `master` branch and saves the revision as a version. To release to `/documentation`, follow these steps:
```